src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Google Inc.
```

Comment:
Is this file supposed to be here? It doesn't seem to be used/related to the picture change.